diff --git a/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js b/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js
index 5d664220c00ab44ea46267508ffd757c6749f6e5..d5803bbad95ad82e182349988a152e8519081875 100644
--- a/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js
+++ b/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js
@@ -3868,7 +3868,7 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (supportsUserTiming && 0 <= childrenEndTime$jscomp$1) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4131,7 +4131,7 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (supportsUserTiming && 0 <= childrenEndTime$jscomp$3) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
diff --git a/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.edge.development.js b/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.edge.development.js
index 66102ed8a0077e9552598f19df59e592d586b551..5effde66ca523a7af8a10552996a6b12836b1724 100644
--- a/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.edge.development.js
+++ b/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.edge.development.js
@@ -4009,7 +4009,7 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (supportsUserTiming && 0 <= childrenEndTime$jscomp$1) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4272,7 +4272,7 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (supportsUserTiming && 0 <= childrenEndTime$jscomp$3) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
diff --git a/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js b/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js
index d9c2e5e801c91b2edbc0101c56af14c3dda6c42e..fcf96b4607ea00b6c925c8ecbf040eff49563b94 100644
--- a/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js
+++ b/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js
@@ -3870,7 +3870,7 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (supportsUserTiming && 0 <= childrenEndTime$jscomp$1) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4133,7 +4133,7 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (supportsUserTiming && 0 <= childrenEndTime$jscomp$3) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
diff --git a/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.edge.development.js b/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.edge.development.js
index 0c02de5c00cd1aecae48ae079cd23e5be2a444b8..22fe8960132e2cbeb42eb29d4aaaff2489fb7bf0 100644
--- a/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.edge.development.js
+++ b/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.edge.development.js
@@ -4013,7 +4013,7 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (supportsUserTiming && 0 <= childrenEndTime$jscomp$1) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4276,7 +4276,7 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (supportsUserTiming && 0 <= childrenEndTime$jscomp$3) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
